.github/workflows/libwasmer-build-shim-linux-arm64.yml

```diff
@@ -0,0 +1,44 @@
+name: libwasmer-build-shim-linux-arm64
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  build:
+    name: Build shim library for Linux ARM64
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: "0"
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Build Docker image
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          load: true
+          platforms: linux/arm64
+          file: ./Docker/arm64.dockerfile
```

Comment:
When this is being built, it will also build the Linux/ARM64 variant of the shim library.